### PR TITLE
Bump version to 1.13

### PR DESCRIPTION
**Pull request by cliu123**
##  opendistro-for-elasticsearch/security pull request intake form
_Please provide as much details as possible to get feedback/acceptance on your PR quickly_

 
 1. __Category:__ (_Enhancement, New feature, Bug fix, Test fix, Refactoring, Maintenance, Documentation_)
 Maintenance
 
 
 2. __Github Issue # or road-map entry, if available:__

 3. __Description of changes:__
Pump version to 1.13

 4. __Why these changes are required?__ 
Prepare for release 1.13

 5. __What is the old behavior before changes and new behavior after changes?__ (_Please add any example/logs/screen-shot if available_)
No changes in behavior

 6. __Testing done:__ (_Please provide details of testing done: Unit testing, integration testing and manual testing_)
 UT
 
 
 7. __TO-DOs, if any:__ (_Please describe pending items and provide Github issues# for each of them_)

 8. __Is it backport from master branch?__ (_If yes, please add backport PR # and commits #_)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by hardik-k-shah — APPROVED** [review]

**Review by debjanibnrj — COMMENTED** [review]
Will there be separate PR to update release notes?

**Review by debjanibnrj — APPROVED** [review]

**Comment by cliu123** [comment]
> Will there be separate PR to update release notes?

@debjanibnrj Yes. There're a bunch of release note items not labeled haven't been categorized by Release Note Drafter. Merging this PR will trigger Release Note Drafter and categorize those items. Release notes will be updated after that.

**Review by palashhedau — COMMENTED** [review]
Is release notes generated for this version?
Ok I read your comment

**Review by palashhedau — DISMISSED** [review]

**Review by vrozov — CHANGES_REQUESTED** [review]
Missing release notes

**Comment by cliu123** [comment]
> Missing release notes

@vrozov  As mentioned above, there're a bunch of release note items not labeled haven't been categorized by Release Note Drafter. Merging this PR will trigger Release Note Drafter and categorize those items. Release notes will be updated after that.
Release notes will be in a seperate PR.

**Comment by vrozov** [comment]
@cliu123 I don't see a need to have a separate PR for release notes. It is also not necessary to merge the PR to trigger github action. I manually re-run  Release Note Drafter, please check release notes template.

File: release-notes/opendistro-for-elasticsearch-security.release-notes-1.13.0.0.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### Enhancements
+
+* Using SAML subject_key and roles_key in the HTTPSamlAuthenticator ([#892](https://github.com/opendistro-for-elasticsearch/security/pull/892))
+* Support for ES system index ([#946](https://github.com/opendistro-for-elasticsearch/security/pull/946))
+* Updating Autheticators to throw RuntimeException on errors ([#505](https://github.com/opendistro-for-elasticsearch/security/pull/505))
+* Add security configuration for Kibana Notebooks ([#903](https://github.com/opendistro-for-elasticsearch/security/pull/903))
+* Short circuit privilege evaluation for bulk requests without index resolution ([#926](https://github.com/opendistro-for-elasticsearch/security/pull/926))
+* Added async search response index to system index list ([#859](https://github.com/opendistro-for-elasticsearch/security/pull/859))
+* Support ES 7.10.2 ([#1005](https://github.com/opendistro-for-elasticsearch/security/pull/1005))
```
Comment (vrozov) [inline release-notes/opendistro-for-elasticsearch-security.release-notes-1.13.0.0.md:13]: Upgrade to patch release should be in Maintenance section.

Comment (cliu123) [reply release-notes/opendistro-for-elasticsearch-security.release-notes-1.13.0.0.md]: Changed.

**Comment by cliu123** [comment]
> @cliu123 I don't see a need to have a separate PR for release notes. It is also not necessary to merge the PR to trigger github action. I manually re-run Release Note Drafter, please check release notes template.

Yes, re-run Release Note Drafter also re-generate the release note draft.

**Comment by cliu123** [comment]
> Missing release notes

@vrozov  Added release notes

**Review by vrozov — APPROVED** [review]

**Review by palashhedau — APPROVED** [review]

**Review by debjanibnrj — APPROVED** [review]

**Pull request merged**